### ISSUE TITLE
[Pytorch/Efficientnet-quant-b0] why I can not get the accuracy 76% for QAT?

### ISSUE THREAD

**Issue by xiaoyizhu** [NVIDIA/DeepLearningExamples#1139]
Related to **Pytorch/Efficientnet QAT** 

**Describe the bug**
why I could not get hte EfficientNetb0 QAT result as described in the webpage?

**To Reproduce**
follow the guidance, download the NGC container, run the QAT training code with offical pretrained EfficientNet-b0 result. After more than 100 epochs, I still could not get the 76.5% accuracy. instead, I can only get aobut 73% accuracy. The only difference from offical guidance is I am using batch size 128. 

**Expected behavior**
I hope I can get accuracy about 76% for efficientnet-quant-b0

**Environment**
using same as mentioned in the pytorch/efficientnet page.
